diff -up wdune-1.908/configure.in.1~ wdune-1.908/configure.in
--- wdune-1.908/configure.in.1~	2020-07-08 16:04:14.000000000 +0200
+++ wdune-1.908/configure.in	2020-07-10 19:30:24.342928272 +0200
@@ -1091,14 +1091,10 @@ fi
 HAVE_OMP_FLAGS=""
 if test "X_$OMP" != "X_no" ; then
  AC_CHECK_HEADER(omp.h,HAVE_OMP_FLAGS=-fopenmp;AC_DEFINE(HAVE_OMP))
- if test "X_$HAVE_OMP_FLAGS" != ""; then
+ if test "X_$HAVE_OMP_FLAGS" != "X_"; then
      CFLAGS="$CFLAGS $HAVE_OMP_FLAGS"
      CXXFLAGS="$CXXFLAGS $HAVE_OMP_FLAGS"
-     if test "X_$CLANG" != "X_"; then
-        export LIBS="$LIBS -fopenmp"
-     else
-        AC_HAVE_LIBRARY(gomp)
-     fi
+     export LIBS="$LIBS -fopenmp"
  fi
 fi
 
diff -up wdune-1.908/src/xerrorhandler.cpp.1~ wdune-1.908/src/xerrorhandler.cpp
--- wdune-1.908/src/xerrorhandler.cpp.1~	2020-07-08 16:04:14.000000000 +0200
+++ wdune-1.908/src/xerrorhandler.cpp	2020-07-10 19:22:39.105312230 +0200
@@ -304,7 +304,7 @@ int DuneApp::emergency_rescue(int sig)
  *  Handler for fatal Xt Errors: try to save files
  */ 
 
-   void fatal_xthandler(char* message)
+   void fatal_xthandler(char* message) __attribute__((noreturn))
       {
       bool exitFlag = true;
       // the following error is (hopefully) not really fatal for white_dune
